README.md

```diff
@@ -28,51 +28,6 @@ growing data sets. Modern APIs allow us to:
-* **GraphQL clients**: clients of the GraphQL API of an ElasticGraph application.
-
-We use the following versioning scheme:
-
-* Version numbers are in a `0.MAJOR.MINOR.PATCH` format. (The `0.` prefix is there in order to reserve `1.0.0` and all later versions
-  for after ElasticGraph has been open-sourced).
-* Increments to the PATCH version indicate that the new release contains no backwards incompatibilities for any stakeholders.
-  It may contain bug fixes, new features, internal refactorings, and dependency upgrades, among other things. You can expect that
-  PATCH level upgrades are always safe--just update the version in your bundle, generate new schema artifacts, and you should be done.
-* Increments to the MINOR version indicate that the new release contains some backwards incompatibilities that may impact the
-  **application maintainers** of some ElasticGraph applications. MINOR releases may include renames to configuration settings,
-  changes to the schema definition API, and new schema definition requirements, among other things. You can expect that MINOR
-  level upgrades can usually be done in 30 minutes or less (usually in a single commit!), with release notes and clear errors
-  from ElasticGraph command line tasks providing guidance on how to upgrade.
-* Increments to the MAJOR version indicate that the new release contains some backwards incompatibilities that may impact the
-  **data publishers** or **GraphQL clients** of some ElasticGraph applications. MAJOR releases may include changes to the GraphQL
-  schema that require careful migration of **GraphQL clients** or changes to how indexing is done that require a dataset to be
-  re-indexed from scratch (e.g. by having **data publishers** republish their data into an ElasticGraph indexer running the new
-  version). You can expect that the release notes will include detailed instructions on how to perform a MAJOR version upgrade.
-
-Deprecation warnings may be included at any of these levels--for example, a PATCH release may contain a deprecation warning
-for a breaking change that may impact **application maintainers** in an upcoming MINOR release, and a MINOR release may
-contain deprecation warnings for breaking changes that may impact **data publishers** or **GraphQL clients** in an upcoming
-MAJOR release.
-
-Each version level is cumulative over the prior levels. That is, a MINOR release may include PATCH-level changes in addition
-to backwards incompatibilities that may impact **application maintainers**. A MAJOR release may include PATCH-level or
-MINOR-level changes in addition to backwards incompatibilities that may impact **data publishers** or **GraphQL clients**.
-
-Note that this policy was first adopted in the `v0.15.1.0` release.  All prior releases aimed (with some occasional mistakes!)
-to follow SemVer with a `0.MAJOR.MINOR.PATCH` versioning scheme.
```

Comment:
I removed this paragraph from `versioning-policy.md`--it's not pertinent to folks anymore.